cmake/modules/Doxyfile.in

```diff
@@ -160,9 +160,9 @@ HTML_TIMESTAMP         = @DOXYGEN_TIMESTAMP@
 # configuration options related to the Qt Docs output
 #---------------------------------------------------------------------------
 GENERATE_QHP       = @GENERATE_QHP@
-QHP_NAMESPACE      = "org.cutelyst.@CUTELYST_VERSION_MAJOR@@CUTELYST_VERSION_MINOR@@CUTELYST_VERSION_PATCH@"
-QHP_VIRTUAL_FOLDER = "cutelyst-@CUTELYST_VERSION_MAJOR@.@CUTELYST_VERSION_MINOR@"
-QCH_FILE           = @CMAKE_BINARY_DIR@/"cutelyst-@CUTELYST_VERSION_MAJOR@@CUTELYST_VERSION_MINOR@@CUTELYST_VERSION_PATCH@.qch"
+QHP_NAMESPACE      = "org.cutelyst.@PROJECT_VERSION_MAJOR@@PROJECT_VERSION_MINOR@@PROJECT_VERSION_PATCH@"
+QHP_VIRTUAL_FOLDER = "cutelyst-@PROJECT_VERSION_MAJOR@.@PROJECT_VERSION_MINOR@"
+QCH_FILE           = @CMAKE_BINARY_DIR@/"cutelyst-@PROJECT_VERSION_MAJOR@@PROJECT_VERSION_MINOR@@PROJECT_VERSION_PATCH@.qch"
```

Comment:
And here...

Reply:
At least for the namespace the Qt convention seems to end it with a dotless version like `org.qt-project.qtcore.561`. For the virtual folder and the file name I can find numerous examples on the web where everybody is doing it in a different way. 🙈 But taking a look into the SQLite data of a Qt qch file shows, that they simply use the project name as folder name. So, because we are going to cutelyst2, I simply use that for the folder and the file name. Think that should also make it easy to have both versions installed the same time.